src/main.cpp

```diff
@@ -38,24 +39,16 @@ int main(int argc, char* argv[]){
 	//////////////////
     bool newSim = true;
     std::string input_file("input.txt");
-    std::string output_directory("result/");
 
     for (int i=0; i<argc; i++) {
         if (strcmp(argv[i],"-i")==0 || strcmp(argv[i], "--input")==0) {
         	input_file = std::string(argv[i+1]);
         }
-        if (strcmp(argv[i],"-o")==0 || strcmp(argv[i], "--output")==0) {
-        	output_directory = std::string(argv[i+1]);
-        	if (output_directory.back() != '/')
-        		output_directory += '/';
-        }
         if (strcmp(argv[i],"-r")==0 || strcmp(argv[i], "--resume")==0) {
             sysinfo.step = atoi(argv[i+1]);
             newSim = false;
         }
     }
-    
-	createOutputDirectory(output_directory);
```

Comment:
This is the place where we should create and read in the `output_directory`. I know that does not work at the moment, because we only create the `ParameterFile` in `setupIC`, but I think we should change that, instead of creating more static variables. So what I have in mind would be something like this here:
```
ParameterFile parameter_file(input_file);
std::cout << "Reading parameters from " << input_file << std::endl;
output_directory = parameter_file.getValueOf("output_directory",std::string("results/"));
if (output_directory.back() != '/')
    output_directory += '/';
```

We would then not hand over the string `input_file` to PROBLEM::setupIC, but instead hand over the object `parameter_file`. Do you think that makes sense?